foundations/installations/installing_node.md

```diff
@@ -43,13 +47,41 @@ command -v nvm
 
 if this returns `nvm: command not found` close the terminal and re-open it.
 
-On OS X you may continue to see the same message. If this happens, open your .bash_profile and add the following line of code:
+In macOS 10.15 and above, the default shell is zsh and nvm will look for .zshrc to update, none is installed by default.
+
+Simply run the following commands:
+
+~~~bash
+touch ~/.zshrc
+~~~
+
+~~~bash
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+~~~
+
+
+
+Paste the following in your terminal and press enter: 
```

Comment:
We should be able to remove this section entirely since it's handled by the installation. This code is also not persistent, and will trick the user into thinking their installation was successful and may mask any issues until they restart their terminal app.

Reply:
Specifically, lines 64-80.

Reply:
Please be more specific 🙏
This section is for mac users who are using zsh

Reply:
Yes, but you *only* need to do this if you don't create the .zshrc file before installing nvm. If nvm finds a .zshrc file, it will automatically add these exports.

See this screenshot for an example. If they want to *immediately* use nvm, they can copy/paste the code, but it's just as easy, if not easier, to get them to close and reopen their terminal. If we include the commands, we should at least advise them *why* they are doing it.

![image](https://user-images.githubusercontent.com/2021268/106882603-d2151980-6743-11eb-96eb-f2c3550c760b.png)